src/pages/docs/guides/chat/build-livestream.mdx

```diff
@@ -0,0 +1,354 @@
+
+It's incredibly rare, but sometimes a client might lose connection to a particular data center. Ably operates in multiple data centers around the world with multiple fallback regions available. If a client can't reach the nearest data center, it will try the next one until the connection is re-established, ensuring minimal downtime and that network issues don't disrupt the experience that you are trying to build.
+
+## Priced for scale
+
+Livestreams can be a high-churn affair, with many users coming and going throughout. In Monthly Active Users (MAU) pricing models, this often leads to bills that far outweigh the actual usage. Ably offers a competitive pricing model that is based on consumption, with discounted rates for higher usage. You won't be billed for users that come and go, only for the time they spend engaging with your application.
+
+If you have an audience that is relatively consistent, for example a number of people sign up for an event and then stay for the duration, Ably also offers a competitive MAU pricing model in addition to the traditional consumption-based model for these scenarios.
+
+A full breakdown of pricing, as well as a cost estimator for your scenario, can be found on the [pricing page](https://ably.com/pricing?chat-type=sales).
+
+### Server-side message batching <a id="server-side-batching"/>
+
+In large livestream chats, the majority of the cost comes from delivering messages to all of the participants. It also limits how many messages per second can be processed.
+
+With server-side batching, Ably groups together messages sent in quick succession on the server and fans them out to clients as one unit. This reduces the total amount of messages sent to each client, greatly reducing your bill. All of this is fully configurable - you can control the rollup period to balance responsiveness in your chat versus cost.
+
+Let's calculate the fanout for a chat room with 1000 users, each posting once per minute:
+
+#### Unbatched scenario:
+
+* 1000 users × 1 message/minute = 1000 messages/minute or 16 messages/second
+* Total fanout: 1000 × 1000 (assuming the message is echoed back to the publisher)
+* Total outbound message rate: **1,000,000 outbound messages/minute** or 16,667 messages/second
+
+#### With 500ms batching:
+
+* Messages are grouped into 500ms batches, so users will see a maximum of 500ms delay from publish to receiving
+* In 500ms, we expect: (1000 messages/minute) × (0.5 seconds/60 seconds) ≈ 8.3 messages per batch
+* Each batch is delivered once to each user
+* Over the space of a minute, there are now only **120,000 outbound messages/minute** or 2000 messages per second
+
+That's an **8x reduction** in the number of messages sent out to clients per minute.
```

Comment:
I'd go with a senario that shows a higher numger, and perhaps explain that as the number of participants grows, theis reduction increases (although perhaps that point is covered sufficiently below)